src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -648,5 +650,13 @@ private boolean checkIfTerrainIsBuildable(Rect2i area) {
         return true;
     }
 
+    /**
+     * This function sets the number of cycles the city must wait before growing. Each cycle usually lasts about 20 seconds.
+     * @param cycles The number of cycles to wait.
+     */
+
+    public void setCityCyclesBeforeGrowth(int cycles){
+        NUMBER_OF_CYCLES_ON_RESET = cycles;
```

Comment:
Since you are setting the value of `NUMBER_OF_CYCLES_ON_RESET` here this cannot be a constant. Thus, it should not be capitalized.